### PR TITLE
Upgrade codecov version to v2

### PR DESCRIPTION
**Pull request by daminijain23**
codecov version was updated to v2 in tox-test.yml as per https://issues.redhat.com/browse/RHELDST-8444

For py38 to work, pyYAML version was downgraded to 5.4.1. pyYAML version 6 was getting installed by default.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 koji
 rpm-py-installer
 requests
-PyYAML
+PyYAML==5.4.1
```
Comment (rohanpm) [inline requirements.txt:4]: This most likely should not be pinned here.  If it has to be pinned for CI to pass, it should be in some file which is only applied during CI.

The problem with adding this here is that this requirements.txt ends up being used in the package metadata.  I think it means you won't be able to install alt-src into a virtualenv along with any other package which also pins a different version of PyYAML.  This would probably also prevent alt-src command from running when installed from an RPM, if the system version of PyYAML doesn't exactly match this.

In summary, if PyYAML has to be pinned that should probably be done via test-requirements.txt or legacy.constraints right now.

However, can you share a bit more info about why you wanted to do this first?  e.g. can you point at a test failure which is being fixed by this?  It might be easier and better to just do a proper fix in the code...

**Review by rohanpm — CHANGES_REQUESTED** [review]

**Comment by daminijain23** [comment]
@rohanpm @rbikar @rajulkumar Rebased this PR and unpinned pyYAML version.
